lte/gateway/c/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -99,8 +98,8 @@ void amf_ue_context_update_coll_keys(
         // context is migrated to map in the upcoming multi-UE PR
         OAILOG_ERROR(
             LOG_AMF_APP,
-            "Insertion of Hash entry failed for  amf_ue_ngap_id "
-            "0x%08" PRIX64 " \n",
+            "Insertion of Hash entry failed for  "
+            "amf_ue_ngap_id " AMF_UE_NGAP_ID_FMT PRIX32 " \n",
```

Comment:
Why do we need PRIX32 here? Please absorb it in the definition of AMF_UE_NGAP_ID_FMT